### ISSUE TITLE
Signal method inserted after the end of C# class implementation in the Godot editor

### ISSUE THREAD

**Issue by aurodev** [godotengine/godot#17849]
Signal method inserted after the end of C# class implementation when connecting a signal
in the Godot editor to a node with a C# script attached.
Presents with error: "A namespace cannot directly contain members such as fields or methods"

Godot version:
3.1.dev.mono.custom_build.b49ca94

OS/device including version:
Windows 8.1 64bit, 8GB AMD FX 6350 6 core

Graphics Chipset
AMD Radeon HD 7700 Series
Memory Size 1024 MB
Radeon Software Version 17.4.4

Issue description:
Signal method inserted after end of the C# class implementation when connecting a signal
in the Godot editor to a node with a C# script attached.
Error: "A namespace cannot directly contain members such as fields or methods"
Just have to drag the signal handler back inside the class brackets.
![godotsignalinsert](https://user-images.githubusercontent.com/7356383/38110770-b580f57e-336a-11e8-930a-a8493484d885.PNG)

Steps to reproduce:
Connect a signal from a node in the editor to a node with a C# script attached.
Signal method inserted after end of class implementation.

**Minimal reproduction project:**

Basic 3D scene such as
https://www.youtube.com/watch?v=-D-IcbsdT04

**Comment by neikeq** [comment]
Related to #12908 (perhaps should be considered a duplicate)

**Comment by karroffel** [comment]
Oh, didn't see that, should probably be marked as duplicate then.

duplicate of #12908

**Comment by aurodev** [comment]
Sorry about the dup, didn't see!

**Comment by vnen** [comment]
Closing as duplicate.

@karroffel that feature worked for me only once, and it doesn't auto-close the issue.
